ui/package.json

```diff
@@ -51,18 +52,13 @@
     "start": "HOST=127.0.0.1 HTTPS=true craco start",
     "build": "craco build",
     "test": "craco test",
-    "lint": "eslint src/",
     "eject": "craco eject",
-    "https_start": "HTTPS=true craco start"
+    "https_start": "HTTPS=true craco start",
+    "lint": "eslint src/ --ext .js --max-warnings=0",
+    "format": "prettier --write \"src/**/*.js\""
   },
   "eslintConfig": {
-    "env": {
-      "browser": true,
-      "node": true
-    },
-    "plugins": [
-      "react"
-    ]
+    "extends": "react-app"
```

Comment:
I am not sure if it shouldn't extend the eslint-config-invenio instead?

Reply:
Thanks! removed!